src/background.js

```diff
@@ -269,9 +276,9 @@ function handleWindowCreated(window) {
   }
 }
 
-function handleWake() {
+function handleWake(alarm) {
   const now = Date.now();
-  log('woke at', now);
+  log('woke at', now, 'with alarm', alarm ? alarm.name : 'none');
```

Comment:
Could we add some telemetry to see if this catches any alarms that wouldn't have fired?
(I think it's a good idea to add, but would like to know how necessary it is…)

Reply:
Hmm, maybe can think of adding an event for late alarms? Will have to refresh my metrics-fu to figure out how to do taht